### PR TITLE
New create actions bookings

### PR DESCRIPTION
**Pull request by 11012022**
Controller bookings works with create 
Simple form of costume show to add dates 
Costume show works 
Bookings save

### CODE REVIEW

File: app/views/costumes/show.html.erb [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
   </div>
 </div>
 
 
-<div class="row mt-5 justify-content-center">
+<div class="book-form justify-content-center">
+<%= simple_form_for [ @costume, @booking ] do |f| %>
  <%= f.input :start_date %>
  <%= f.input :end_date %>
+ <%= f.submit "Book stume", data: { confirm: "Are you sure you want to book this Costume?"},  class: "btn btn-primary" %>
```
Comment (Corentr1) [inline app/views/costumes/show.html.erb]: need to change the alert

**Comment by Corentr1** [comment]
LGTM

**Comment by ChloePeter** [comment]
C'est cool

**Comment by JonnySomi** [comment]
Nice work, LGTM

**Pull request merged**